csc_335_restaurant/src/model/Server.java

```diff
@@ -10,54 +10,46 @@
 import java.util.ArrayList;
 
 public class Server {
-	// private ArrayList<Table> tables;
+	private ArrayList<Integer> tableNums;
 	private String name;
-	private int num_seated;
 	private double tips;
 	
 	public Server(String name) {
-		//tables = new ArrayList<Table>();
+		this.tableNums = new ArrayList<Integer>();
 		this.name = name;
-		num_seated = 0;
-		tips = 0.0;
+		this.tips = 0.0;
 	}
 	
-	/*
-	public void seatTable(Tabel t) {
-		// when every table is sat num_seated is updated to add the num of seats from that table
-		tables.add(t);
-		num_seated += t.getSeats();
+	public void addTable(int table) {
+		if (!tableNums.contains(table)) {
+		    tableNums.add(table);
+		}
```

Comment:
Just to ensure consistency
```suggestion
	public void addTable(int tableNum) {
		if (!tableNums.contains(tableNum)) {
		    tableNums.add(tableNum);
		}
```